Pages/Shared/layout.html

```diff
@@ -1,22 +1,38 @@
+<!doctype html>
 <html>
 <head>
     <title>Template</title>
-    <script src="../../Libraries/jquery-3.5.1.min.js"></script>
-    <script>
-        function check() {
-            $('#tar').load('error.html');  // here 'test.html' is a page and 'target' id a id of 'test.html'
```

Comment:
tar is not a valid name. Please add valid names. Also, do not just copy paste the code. Remove all unwanted code and comments.